scripts/compile-triton.sh

```diff
@@ -184,7 +184,7 @@ build_triton() {
   pip install -vvv -e '.[build,tests]'
 
   # Copy compile_commands.json in the build directory (so that cland vscode plugin can find it).
-  cp $TRITON_PROJ_BUILD/"$(ls $TRITON_PROJ_BUILD)"/compile_commands.json $TRITON_PROJ/
+  cp $(find $TRITON_PROJ_BUILD -name compile_commands.json) $TRITON_PROJ/
```

Comment:
This should also work:
```suggestion
  cp $TRITON_PROJ_BUILD/*/compile_commands.json $TRITON_PROJ/
```

Reply:
The current implementation is the same as in readme: `Get the full path to the compile_commands.json file produced by the build: find python/build -name 'compile_commands.json' | xargs readlink -f.` but your version is more laconic :)